_layouts/default.html

```diff
@@ -4,15 +4,18 @@
 
   <body>
 
-    <div>
-      <a href="{{ site.baseurl }}/">
-        <header>
-          <img src="{{ site.baseurl }}/img/logo.svg" alt="{{ site.title }}" />
-        </header>
-      </a>
-
-      {{ content }}
-
+    <div id="wrapper">
+      <div id="content">
+        <div>
```

Comment:
Ok.
Maybe you can use `<main>` rather than `id="content"` and some `<section>`?  😃